### DISCUSSION TITLE
storage space on different machines

### DISCUSSION THREAD

**Discussion by mahdipsycho** [Chia-Network/chia-blockchain#4338] [Q&A]
hey everyone
I was just wondering that is there any different between 1 machine that has 100 TB HDD or 5 machines tha have 20TB HDD on same wallet
I mean as chiastatus says if you have 10TB plots you must wait 3 Months to mine a block 
and thats so much time
so i want to know if i have 100 TB hdd on 5 machine I would mine a block in 9 days(if difficulty stay the same)
or I will mine 10 blocks in 90 days?
I would be thankful if you guys read this and answer me
thanks everyone for your time

**Comment by mahdipsycho** [comment]
I mean
I run two full nodes and dont know how to set harvester 
and BTW they are not on same network

**Comment by ilyakooo0** [comment]
The major factor is the total space that your *different* plots use. (i e copying the same plots multiple times does absolutely nothing). Theoretically splitting space between multiple machines could slightly improve access time (accessing 20 TB would be faster than accessing 100 TB), but I doubt this would make a real difference in practice.

Bottom line: it doesn't matter if you split space between machines.

**Reply by mahdipsycho** [reply]
thanks for the answer 
But I wanna know is it the same if I sync the blockchain on every machine or make a machine as main machine and have the wallet and keys and the others be harvester 
cause I cant make harvesters
